src/components/pill/components/button-pill/config/index.js

```diff
@@ -0,0 +1,24 @@
+import { PILL_TYPE } from '../../../enums';
+import { CATEGORIES } from '../../../../../../.storybook/utils.js';
+
+export const args = {
+  type: '',
+};
+
+export const argTypes = {
```

Comment:
In `vlPillElement` worden meer attributes gedefinieerd dan enkel type, hoe zit dat dan juist?

Reply:
Klopt, Dat is omdat vlPillElement enkel het `type` attribuut gaat observer voor het element zelf. 
Bij de gewone pill wordt ook bv. `disabled` observed maar bijd e child attributen dus dat is hier niet van toepassing. Enkel type werkt bij de button in dit geval.